controllers/operands/kubevirt.go

```diff
@@ -669,13 +673,34 @@ func hcTLSSecurityProfileToKv(profile *openshiftconfigv1.TLSSecurityProfile) *ku
 	}
 }
 
-func getKVDevConfig(hc *hcov1beta1.HyperConverged) *kubevirtcorev1.DeveloperConfiguration {
+func hcMemoryOvercommitToKv(spec *hcov1beta1.HyperConvergedSpec) (percent int, err error) {
+	if spec.HigherWorkloadDensity == nil {
```

Comment:
This field has actually to purposes: one is to hold the feature configurations and one is to enable/disable the feature. I think is is wrong, mostly because personally, I'm really prefer explicit APIs, but also because it's confusing and error prompt. 

In addition, there is a bug in the UI that auto populate optional fields, making this feature to be always active, if creating the HyperConverged CR from the UI.

What do you think about adding a boolean field for that?

Reply:
@nunnatsa Thanks! you are correct. I am going to change the API a bit - the field won't optional anymore, it will have a defualt of 1, exactly as it is in kubevirt. And the UI will set it to 1.5 automatically as the user will opt in for memory overcommit.